src/nexuscli/cli/subcommand_repository.py

```diff
@@ -64,6 +76,10 @@
   --remote_auth_type=<remote_auth_type> Accepted: username [default: None]
   --remote_username=<remote_username>   Remote username
   --remote_password=<remote_password>   Remote password
+  --gpg=<gpg-file-path>                 gpg file [default: Release.key]
```

Comment:
The [Nexus documentation](https://help.sonatype.com/repomanager3/formats/apt-repositories#AptRepositories-HostingAptRepositories) refers to this file as `public.gpg.key`.

Can we please default this to the same name for clarity?  I'd also use `./` to be explicit on the path

```suggestion
  --gpg=<gpg-file-path>                 gpg file [default: ./public.gpg.key]
```